app/controllers/account/pages_controller.rb

```diff
@@ -20,8 +20,30 @@ def rate
     end
   end
 
+  def new
+    @page = Page.new
+    @company = parent
+  end
+
+  def create
+    @pages = collection
```

Comment:
no need to create separate variable for this

Reply:
it does not see "collection" at view part

Reply:
why'd you need a 'collection' within a view?